students/20th/junmopark/main.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+        integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+        crossorigin="anonymous" />
+    <title>Instagram</title>
+</head>
+<body>
+    <section>
+        <nav class="topNav">
+            <header class="navTopItem">
+                <h1 class="title">Instagram</h1>
+            </header>
+            <div class="navTopItem">
+                <input type="text" class="search" placeholder="검색">
+            </div>
+            <div class="navTopItem">
+                <img alt="quest" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png">
+                <img alt="like" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png">
+                <img alt="profile" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png">
+            </div>
```

Comment:
이 부분은 어떤 이미지들의 list니까,
div > li 로 처리하기보다는
ul > li로 표시하는게 좀 더 semantic해보이네요!
```suggestion
            <ul class="navTopItem">
              <li>
                 <img alt="quest" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png">
              </li>
              <li>
                 <img alt="quest" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png">
              </li>
              <li>
                 <img alt="quest" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png">
              </li>
            </ul>
```

Reply:
수정했습니다!